packages/core/src/memo.ts

```diff
@@ -0,0 +1,29 @@
+import type { StateNode } from '.';
+
+const stateNodeMap = new WeakMap<StateNode, any>();
+
+export function getMemo<T>(
+  stateNode: StateNode<any, any>,
+  key: string
+): T | undefined {
+  return stateNodeMap.get(stateNode)?.[key];
+}
```

Comment:
this one stays unused, right? it's simple enough that we can always add it when the need arise - so I would recommend dropping it for the time being if it truly stays unused

Reply:
You're right; it was used during the refactor and then I removed it's only usage 😜

Reply:
Removed